src/sentry/snuba/outcomes.py

```diff
@@ -226,11 +226,13 @@ def __init__(
         raw_groupby = query.getlist("groupBy", [])
         if len(raw_fields) == 0:
             raise InvalidField('At least one "field" is required.')
-
+        isProject = False
+        if "project" in raw_groupby:
+            isProject = True
```

Comment:
Python conventions generally use `snake_case`

```suggestion
            is_project = True
```